protoc_plugin/plugin.py

```diff
@@ -106,7 +106,7 @@ def render(
                         raise TypeError(cardinality)
 
                     original_method = f"grpclib_stub.{name}"
-                    buf.add(f"self.{name} = {wrapper_cls}({original_method}, client)")
+                    buf.add(f"self.{name} = {wrapper_cls}({original_method}, client, grpclib_stub_attr)")
```

Comment:
Couldn't we just pass the `grpclib_stub` directly to the wrapper_cls intead of the name of the attribute?

Reply:
Because of this stuff:

https://github.com/modal-labs/modal-client/blob/main/modal/client.py#L307-L325

ie. if the process is forked, we might want to re-establish connections and create new stubs, and retaining a reference to the old stub would likely break in many different ways as it would still be using the gRPC channel from the parent process.